packages/helix/helix.pacscript

```diff
@@ -0,0 +1,36 @@
+name="helix"
+version="22.08.1"
+repology=("project: helix")
+url="https://github.com/helix-editor/helix/archive/refs/tags/${version}.tar.gz"
+build_depends="cargo git"
+description="A post-modern modal text editor"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+hash="c9b184d53bab51e7a9fe81dc8c191a853177abba67ea408e968937f31b50b45a"
+
+prepare() {
+    true
+}
+
+build() {
+    cargo build -j$(nproc) --release --locked
+}
+
+install() {
+    sudo install -Dm755 target/release/hx -t "${pkgdir}/usr/lib/${name}"
+
+    echo '#!/bin/sh\nHELIX_RUNTIME=/usr/lib/helix/runtime exec /usr/lib/helix/hx "$@"' > hx
```

Comment:
I don't think newlines work on single quotes

Reply:
The file ends up correct when I tested the package